docs/fleet/fragments.md

```diff
@@ -175,170 +130,177 @@ This example assumes the fragment with ID `abcd7ef8-fa88-1234-b9a1-123z987e55aa`
 5. Add any update operators you'd like to apply to the fragment to the `mods` section.
    Click to view each example:
 
-   {{< expand "Change the name and attributes of a component" >}}
-   This example uses [`$set`](https://www.mongodb.com/docs/manual/reference/operator/update/set/#mongodb-update-up.-set) to make the following changes to the attributes of a motor named `motor1`:
+{{< expand "Change the name and attributes of a component" >}}
```

Comment:
Note to reviewers: I didn't change any of the expander content, just un-indented to fix formatting bug after putting this into tabbed content